File: python/01-commands/doc/Banking_Kata/banking_transaction_dto.py
from datetime import datetime

class BankingTransactionDTO:
    def __init__(self, date, amount, balance):
        self._date = date
        self._amount = amount
        self._balance = balance

    @property
    def date(self):
        return self._date

    @property
    def amount(self):
        return self._amount

    @property
    def balance(self):
        return self._balance

File: dotnet/01-commands/doc/Banking_Kata/BankingTransactionDTO.cs
namespace Banking_Kata
{
    public record BankingTransactionDTO
    {
        public DateTime date { get; private set; }
        public int amount { get; private set; }
        public int balance { get; private set; }

        public BankingTransactionDTO(DateTime date, int amount, int balance)
        {
            this.date = date;
            this.amount = amount;
            this.balance = balance;
        }
    }
}